app/controllers/concerns/application_shared.rb

```diff
@@ -15,4 +15,12 @@ def set_timezone
     # Reset timezone for the next thread
     Time.zone = default_timezone
   end
-end
\ No newline at end of file
+
+  private
+
```

Comment:
why go method missing path instead of dynamic method creation? usually method_missing has a performance penalty.